balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -261,6 +290,59 @@ def update_latest():
 
         retry(update_latest, sleeptime=2, max_sleeptime=2, attempts=10)
 
+    def run_backend2(
+        self, platform, buildID, productName, branch, appVersion, locale, hashFunction, extVersion, schemaVersion, isOSUpdate=None, **updateKwargs
+    ):
+        log.info("Using backend version 2...")
+        balrog_api = functools.partial(do_balrog_req, auth0_secrets=self.auth0_secrets)
+
+        targets = buildbot2updatePlatforms(platform)
+        build_target = targets[0]
+        alias = None
+        if len(targets) > 1:
+            alias = targets[1:]
+            log.debug("alias entry of %s ignored...", json.dumps(alias))
+        data = {"buildID": buildID, "appVersion": appVersion, "platformVersion": extVersion, "displayVersion": appVersion}
+        if isOSUpdate:
```

Comment:
Please remove this; it was only ever used for B2G. It's a good time to kill the cruft.